jetbrains-core/src/software/aws/toolkits/jetbrains/ui/ResourceSelector.kt

```diff
@@ -81,15 +83,16 @@ class ResourceSelector<T> : ComboBox<T>() {
                 null
             } ?: return@executeOnPooledThread
 
-            ApplicationManager.getApplication().invokeLater({
+            runInEdt(ModalityState.any()) {
```

Comment:
maybe we should have a ktlint rule here if we want to be consistent around how we run on the UI thread (e.g. `ApplicationManager` vs simple `runInEdt`

Reply:
I tried adding a BannedPattern in the existing BannedPatternRule but it didn't work. ktlint doesn't seem to even catch the existing banned patterns defined in this rule. I tried using the existing banned patterns in my code and reran ktlint, it succeeds.